config.yaml

```diff
@@ -5,23 +5,23 @@ results_dir: results/
 summary_dir: results/
 costs_dir: data/ #TODO change to the equivalent of technology data
 
-run: test_shipping_nTESTTEST_GGGGGGGGGGG
+run: Runname_test
 
 foresight: overnight
 
 scenario:
   simpl: # only relevant for PyPSA-Eur
   - ""
   clusters: # number of nodes in Europe, any integer between 37 (1 node per country-zone) and several hundred
-  - 9077
+  - 90774
```

Comment:
Just out of interest: Is there any reason for this extremly high number of nodes?